pkg/utils/command.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2021 Chaos Mesh Authors.
```

Comment:
How about using a single pkg instead of `utils`?

Reply:
Fixed.